addons/stock/models/stock_picking.py

```diff
@@ -919,14 +920,12 @@ def _log_activity(self, render_method, documents):
         """
         for (parent, responsible), rendering_context in documents.items():
             note = render_method(rendering_context)
-
-            self.env['mail.activity'].create({
-                'activity_type_id': self.env.ref('mail.mail_activity_data_warning').id,
-                'note': note,
-                'user_id': responsible.id,
-                'res_id': parent.id,
-                'res_model_id': self.env['ir.model'].search([('model', '=', parent._name)], limit=1).id,
-            })
+            parent.activity_schedule(
+                'mail.mail_activity_data_warning',
+                date.today(),
```

Comment:
You should use [fields.Date.context_today()](https://github.com/odoo/odoo/blob/937d870ffc42a789ec9a3d38cb317a8432b04191/odoo/fields.py#L1544)